frontend/src/TourForm/TourForm.tsx

```diff
@@ -119,11 +131,15 @@ function TourForm({ handleCancel }: ITourFormProps) {
    * Filters through murals to determine what should appear in autocomplete
    */
   useEffect(() => {
-    const filtered = murals.filter((mural: any) => {
-      return mural.name.toLowerCase().includes(muralQuery);
-    });
-    setMuralResults(filtered);
-  }, [muralQuery, murals]);
+    if (muralQuery) {
```

Comment:
how come this logic is being added now? I'm not sure I understand what it does

Reply:
Oh, I needed to add that because if there isn’t a query passed in, no murals would be returned, lmk if it’s ok!